src/routes/ui/templates.rs

```diff
@@ -5,6 +5,89 @@ use rocket::request::FlashMessage;
 use routes::ui::view_helpers::*;
 use std::ops::Deref;
 
+macro_rules! HtmlTemplate {
+    ($x:tt) => {{
+        $x {
+            _parent: BaseTemplate {
+                revision: ::GIT_REV,
```

Comment:
note: `::GIT_REV` should _technically_ be `$crate::GIT_REV`, but it's probably fine since we're not using `HtmlTemplate` and friends outside of the `rustodon` crate.

Reply:
/That's/ what I was looking for, I kept on trying various sequences of `super` and `self`, but found the unqualified lookup got me what I wanted. Will switch to `$crate` anyways because that's what I wanted.

Reply:
(Done)